app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+class SessionsController < ApplicationController
+  skip_before_action :authenticate_user!, only: %i[new create]
+  skip_before_action :check_user_active, only: %i[new create]
+  def new; end
+
+  def create
+    user = find_user
+    if user_authenticated?(user)
+      login_user(user)
+    else
+      handle_invalid_credentials
+    end
+  end
+
+  def destroy
+    session[:user_id] = nil
+    redirect_to root_path, notice: 'Logged out successfully'
```

Comment:
I see you have a controller test for #destroy, but Coveralls is saying that the two lines in your destroy method are not covered for some reason.